code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -833,6 +833,47 @@
 	new /obj/item/weapon/bonegel(location)
 	qdel(holder.my_atom)
 
+/datum/chemical_reaction/bonemilk
+	name = "Bone fixer"
+	id = BONEFIXER
+	result = BONEFIXER
+	required_reagents = list(MILK = 10, CLONEXADONE = 10, PHOSPHORUS = 10)
+	result_amount = 5
+
+/datum/chemical_reaction/bonezone
+	name = "Calcium-plasteel reinforcement"
+	id = BONEZONE
+	result = BONEZONE
+	required_reagents = list(BONEFIXER = 10, IRON = 30, PLASMA = 10)
+	result_amount = 5
+
+/datum/chemical_reaction/bonezone/on_reaction(var/datum/reagents/holder)
```

Comment:
Please document what this whole thing does for the changelog.

It appears that, when you make bonezone not inside a mob (e.g.: in a chem dispenser or beaker) and there is still 5u bonefixer left over afterward, there's a 1/20 chance to spawn smoke, play a song, and a make a new object.

The object created is 1/20 chance to be a skeleton monkey (plasma skeleton monkey if you had 20u plasma) and otherwise just a skull.

Reply:
yes, a little silly flavour stuff, keeps the chemists guessing